### PR TITLE
[FIX][4.0] An error has occurred on Search Filter

### PR DESCRIPTION
**Pull request by amitranjan2**
Pull Request for Issue #23677

### Summary of Changes
Added
use Joomla\CMS\Language\Text;
use Joomla\CMS\Factory;
use Joomla\CMS\Router\Route;
use Joomla\CMS\Layout\LayoutHelper;

and changed JText to Text 
similary to JFactory ,JRoute,JLayoutHelper

### Testing Instructions
Components >> smart search >> Search Filters >>New
then write title name and click >> save and close

### Expected result

It should be saved and title name should be displaced

### Actual result

0 Class 'Text' not found

### Documentation Changes Required
NO

### CODE REVIEW

File: administrator/components/com_finder/tmpl/filters/default.php [hunk 1]
```diff
@@ -10,26 +10,30 @@
 defined('_JEXEC') or die;
 
 use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
```
Comment (Quy) [inline administrator/components/com_finder/tmpl/filters/default.php:14]: Please sort in alpha order.

Comment (amitranjan2) [reply administrator/components/com_finder/tmpl/filters/default.php:14]: Done

File: administrator/components/com_finder/tmpl/filters/default.php [hunk 2]
```diff
@@ -9,27 +9,31 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Factory;
 use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Layout\LayoutHelper
```
Comment (Quy) [inline administrator/components/com_finder/tmpl/filters/default.php]:
```suggestion
use Joomla\CMS\Layout\LayoutHelper;
```

**Comment by amitranjan2** [comment]
@Quy I think it's fine now, please review this

**Comment by anuragteapot** [comment]
I have tested this item :white_check_mark: successfully on f00491ac1a62df2edf8529605f9664c5a3939e19<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23679">issues.joomla.org/tracker/joomla-cms/23679</a>.</sub>

**Comment by amitranjan2** [comment]
@Quy could you please test this PR ?

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23679">issues.joomla.org/tracker/joomla-cms/23679</a>.</sub>

**Comment by amitranjan2** [comment]
@wilsonge  if no issue , please merge this

**Comment by HLeithner** [comment]
thx

**Comment by amitranjan2** [comment]
thanks

**Pull request merged**